src/style.css

```diff
@@ -8,12 +8,13 @@ body {
 }
 
 #parent {
-  width: 35%;
-  max-height: 500px;
+  width: 50%;
+
+  /* min-height: 500px; */
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: